test/NuGet.Core.FuncTests/NuGet.XPlat.FuncTest/XPlatClientCertTests.cs

```diff
@@ -124,6 +124,7 @@ public void ClientCertAddCommand_Fail_NoSourceSpecified()
             Assert.Equal(1, exitCode);
         }
 
+        // https://github.com/NuGet/Home/issues/9684
```

Comment:
I'm not sure why we are adding this here?

Reply:
They're skipped for the same reason (failing on mac and/or linux). The linked issue explains a little better.  We should have documented it when we skipped the tests in the first place.